config/settings.js

```diff
@@ -27,11 +35,19 @@
  */
 
 window.spinnakerSettings = {
+  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
```

Comment:
Actually, looking at deck, I think this is just a fallback list in case no cloud providers are enabled. It seems like this can probably just be removed.

See [code](https://github.com/spinnaker/deck/blob/fee4c9839a465a614c105a52267e3ccb9f0e63e5/app/scripts/modules/core/account/account.service.js#L71) and [test](https://github.com/spinnaker/deck/blob/0a9ed6034363ee8c1c75958822b4e9009f6781b9/app/scripts/modules/core/account/account.service.spec.js#L130).